docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -905,14 +909,29 @@ sources described above to create conditional samplers for first name
 and last name based on soundex.  Perhaps measure of success is to look
 at entropy of character n-gram distribution.
 
-To simulate naming after a parent or family member, have 5% of children 
-born in the simulation be named after a relative that matches their 
-sex. To do this: 
+To simulate naming after a parent or family member, we would like 5% of 
+children to have the same name as a parent. This can be separated into two 
+groups: 
+
+**People born in the simulation**
+
+For 5% of simulants, they will be assigned a name based on these steps: 
 
-#. If there is a tracked parent of the same sex, assign the same first name 
-#. If there is a relative of the same sex, assign the same first name. Note that this might be a relative of the reference person (i.e., a male child has a tracked mother, but that tracked mother has a tracked father) 
+#. Female simulants will have the same last name as their mother (who is known) 
+#. Male simulants, if the mother is the reference person and has an opposite-sex spouse, they will receive that name 
+#. Otherwise, if the mother has a male relative known in the simulation (either the mother is the reference person or is related to the reference person), they will be assigned a males relative's name. The relative can be selected at random. 
 #. If none of these are available, assign a random name 
 
+**People initialized in the simulation**
+
+For 5% of simulants, they will be assigned a name based on these steps: 
```

Comment:
To me, it feels fairly counter-intuitive to select the 5% of simulants *first* and then check if they can receive a matched name. But, I think it is mathematically correct.

Reply:
Yeah, I see what you mean. We will end up with less than 5% overall, but since the 5% was arbitrary anyways, I am not too concerned if we don't hit our made up target